test/integration/stats_integration_test.cc

```diff
@@ -213,8 +213,9 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithStats) {
   // 2019/05/07  6794     49957               Stats for excluded hosts in cluster
   // 2019/04/27  6733     50213               Use SymbolTable API for HTTP codes
   // 2019/05/31  6866     50157               libstdc++ upgrade in CI
+  // 2019/06/03  7146     49137               absl update
 
-  EXPECT_EQ(m_per_cluster, 50157);
+  EXPECT_EQ(m_per_cluster, 49137);
```

Comment:
nice reduction of almost 1k per cluster. I'm surprised absl made that much of a difference. I assume this is not because o of the service node_hash_map, which I don't think would be replicated per cluster.

Reply:
The reduction was before I made node_hash_map change I think.